GOVERNANCE.md

```diff
@@ -108,23 +108,14 @@ Please also make a note in the change log under [`# Governance history`](#govern
 
 # Finances and Funding
 
-There is currently no mechanism for the data.table project to receive funding as an entity.  
+The data.table project is fiscally sponsored by [NumFOCUS](https://numfocus.org/).  Donations to the project can be made at [numfocus.org/donate-to-data-table](https://numfocus.org/donate-to-data-table)
```

Comment:
I don't really know how NumFOCUS funding works, but as long as there are no funds coming from it, I would phrase it "project can be fiscally sponsored via NumFOCUS". If the

Reply:
Saying that "project is [already] fiscally sponsored" is a bit more discouraging new potential sponsors, comparing to saying "project can be sponsored". Ultimately I don't have strong opinion on that, let's see what others think.

Reply:
This language came directly from NumFOCUS, but I agree with Jan's take - we'll ask them in our meeting next week if the language can be tweaked to clarify in some way that we receive sponsorship _through_ NumFOCUS.